### DISCUSSION TITLE
Build time transforms?

### DISCUSSION THREAD

**Discussion by AkimaLunar** [microsoft/griffel#143] [Q&A] [answered]
Do you have anything ready for [build time transforms](https://griffel.js.org/react/guides/build-time-transforms)? If you do, but haven’t added the docs yet, I can contribute.

Is [this #139 ](https://github.com/microsoft/griffel/pull/139) related perhaps? 😄

**Comment by layershifter** [comment, marked as answer]
Hey, yes we have:

- https://github.com/microsoft/griffel/tree/main/packages/babel-preset
- https://github.com/microsoft/griffel/tree/main/packages/webpack-loader

There are even some docs in Fluent UI: https://react.fluentui.dev/?path=/docs/concepts-developer-build-time-styles--page

> If you do, but haven’t added the docs yet, I can contribute.

It would be great ❤️

**Reply by AkimaLunar** [reply]
Perfect 👌
